pkg/core/xds/matched_policies.go

```diff
@@ -14,11 +14,9 @@ type MatchingPolicyMap map[core_model.ResourceType][]core_model.Resource
 // TypedMatchingPolicies all policies of this type matching
 type TypedMatchingPolicies struct {
 	Type              core_model.ResourceType
-	InboundPolicies   map[mesh_proto.InboundInterface][]core_model.Resource
-	OutboundPolicies  map[mesh_proto.OutboundInterface][]core_model.Resource
-	ServicePolicies   map[ServiceName][]core_model.Resource
```

Comment:
I don't think you should remove it yet. OPA for example uses this

Reply:
I think OPA uses `CustomInboundPolicies` https://github.com/kumahq/kuma/blob/master/pkg/core/xds/matched_policies.go#L27

Reply:
Not with: https://github.com/Kong/kong-mesh/pull/1743/files Which intends to remove the old hacky implem

Reply:
👍